homepage/styles.css

```diff
@@ -0,0 +1,142 @@
+html {
+    background-color: grey;
+}
+body {
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+.header {
+    width: 100%;
+    height: 500px;
+    color: white;
+    background-image: url("background.jpg");
+    background-repeat: no repeat;
+    background-size: cover;
+    border: 1px solid green;  
+} 
+.nav {
+    height: 50px;
+    width: 100%;
+    margin-bottom: 5px;
+    border-bottom: 0.5px solid lightgrey;
+    padding-top: 15px;
+    padding-bottom: 15px;
+    font-family: 'Times New Roman', Times, serif;
+    font-size: 14px; 
+    }
+
+ul {
+  list-style-type: none;      
+  
+}
+li {
+    margin: 0 5px;
+    padding-right: 10px;
+    
+}
+a{
+    color: white;
+    text-decoration-line: none;    
```

Comment:
Check the browser compatibility here.
https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration-line

What it does differently than just `text-decoration: none;` here?